concordium-sdk/src/main/java/com/concordium/sdk/Client.java

```diff
@@ -226,8 +226,22 @@ public Hash sendTransaction(Transaction transaction) throws TransactionRejection
         throw TransactionRejectionException.from(transaction);
     }
 
+    public <P extends IAccountTransactionPayload2> boolean sendTransaction2(
```

Comment:
**Should we shift to a Generic `SendTransaction` method?**

User of the code / SDK would directly be able to figure out what are the types of Payloads supported.